functions/v2/ocr/ocr-translate-text/src/main/java/functions/OcrTranslateApiMessage.java

```diff
@@ -59,7 +59,8 @@ public String getLang() {
     return lang;
   }
 
-  public static OcrTranslateApiMessage fromPubsubData(byte[] data) {
+  @SuppressWarnings("unchecked")
```

Comment:
Extra spaces before many of the the "@SuppressWarnings" annotations

Reply:
Good catch!